ngshare/test_ngshare.py

```diff
@@ -77,6 +77,21 @@ def assert_success(url, data=None, params=None, method='GET'):
     return resp
 
 
+@pytest.mark.gen_test
+def test_health(http_client, base_url):
+    'Test /healthz endpoint'
+    response = yield http_client.fetch(base_url + '/healthz')
```

Comment:
You can use `assert_success()` if you would like.

Reply:
I didn't want to use `server_communication` since k8s won't query the endpoint with a user parameter.